src/python/pants/engine/internals/build_files.py

```diff
@@ -229,17 +306,24 @@ async def parse_address_family(
         dependents_rules_parser_state = None
         dependencies_rules_parser_state = None
 
+    # ISSUE, want to use the MultiGet variant here...
+    all_env_vars = itertools.cycle((EnvironmentVars({}),))
+    # all_env_vars = await MultiGet(
+    #     Get(BUILDFileEnvironmentVars, BUILDFileEnvironmentVarsRequest(fc)) for fc in digest_contents
+    # )
```

Comment:
This did PASS the tests, so with limited rule graph it works.. but it fails when used along with the rest of the pants world.